virttest/utils_hotplug.py

```diff
@@ -483,3 +486,44 @@ def vcpuhotunplug_unsupport_str():
         return "not currently supported"
     else:
         return ""
+
+
+def create_mem_xml(params):
+    """
+    Create memory device xml.
+
+    :param params: Dictionary containing the test parameters.
+    :return: Returns a copy of instance not sharing any
+             references or modifications
+    """
+    mem_xml = memory.Memory()
+    mem_xml.mem_model = params.get("mem_model", "dimm")
+    tg_size = params.get("tg_size")
+    tg_sizeunit = params.get("tg_sizeunit", 'KiB')
+    tg_node = params.get("tg_node", 0)
+    pg_size = params.get("page_size")
+    pg_unit = params.get("page_unit", "KiB")
+    node_mask = params.get("node_mask", "0")
+    mem_addr = ast.literal_eval(params.get("memory_addr", "{}"))
+    numa_cells = params.get("numa_cells", "").split()
+
+    if tg_size:
+        tg_xml = memory.Memory.Target()
+        tg_xml.size = int(tg_size)
+        tg_xml.size_unit = tg_sizeunit
+        # There is support for non-numa node
+        if numa_cells:
```

Comment:
`numa_cells` does not have much significance, can we avoid it, if tg_xml.node is optional we can just omit to update.

Reply:
I have used in hotplug tests.

Reply:
it will help to make the function more generic.